source/index.rst

```diff
@@ -1,6 +1,18 @@
 地震学入门教程
 ##############
 
+.. toctree::
+   :maxdepth: 3
+   :caption: 编程基础
+   :hidden:
+
+   programming/introduction
+   programming/bash
+   programming/python
+   programming/c
+   programming/fortran
+   programming/compile
+   programming/git
```

Comment:
rename the file from git to version-control?

Reply:
Done.